src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -136,4 +136,51 @@ private string GetIconName()
             }
         }
     }
+
+    internal class ReflectionZeroTouhSearchElement : NodeSearchElement
+    {
+        private readonly ReflectionFunctionDescriptor functionDescriptor;
+        private readonly string fullname;
+        public override string CreationName { get { return functionDescriptor != null ? functionDescriptor.MangledName : this.Name; } }
+
+        public override string FullName { get { return fullname; } }
```

Comment:
```suggestion
        public override string FullName => fullname;
```
I prefer the arrow getter, but independently of which option should be consistent with the rest of the code (`Descriptor` uses arrow while this `get` and `return`)